src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx

```diff
@@ -153,7 +153,7 @@ Type 'help' to get help.</value>
                   [-EncodedCommand &lt;Base64EncodedCommand&gt;]
                   [-ExecutionPolicy &lt;ExecutionPolicy&gt;] [-InputFormat {Text | XML}]
                   [-Interactive] [-MTA] [-NoExit] [-NoLogo] [-NonInteractive] [-NoProfile]
-                  [-OutputFormat {Text | XML}] [-SettingsFile &lt;filePath&gt;] [-STA] [-Version]
+                  [-OutputFormat {Text | XML}] [-SettingsFile &lt;filePath&gt;] [-SSHServerMode] [-STA] [-Version]
```

Comment:
I don't think we should document the flags that are used solely for remoting scenarios (starting pwsh as a backend server). The related flags are: `-socketservermode`, `-servermode`, `namedpipeservermode`, and `-sshservermode`.

They are for remoting scenarios, and is not useful to an end-user.

Reply:
`-sshservermode` is the exception. We document using it in the SSH remoting article. Since it is exposed to users in that article, we need to document it. We have had several questions about where the documentation is for that parameter.